docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -818,210 +420,267 @@
+    "1. If the initialised `database` does not contain an `experiment`, then the instantiaion will raise an error and halt your work.  \n",
+    "2. If the database already contains an `experiment`, then the instantiated `measurement`  will be added to the most recent `experiment` in the database without raising an error message or warning. This will lead to poor data management.\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Load or create experiment"
+    "### Registering parameters to measure\n",
+    "\n",
+    "QCoDeS features the ability to store the relationship between parameters (i.e. parameter `y` is dependent on `x`). This feature allows the intent of the measurement to be clearly recorded in the expriemental records. In addition, the parameter dependency is used to define the coordinate axes when plotting the data using QCoDeS. The parameters which are being measured are first registered with the `measurement`. When registering a dependent parameter (i.e. y(x)) the independent parameter is declared as a setpoint. As a consequence, independent parameters must be registered prior to their corresponding dependent parameters. \n",
+    "\n",
+    "In our example, ``dac.ch1`` is the independent parameter and ``dmm.v1`` is the dependent parameter. So we register ``dmm.v1`` with the setpoint as ``dac.ch1``. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qcodes.dataset.measurements.Measurement at 0x20b3f7426a0>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "context_meas.register_parameter(dac.ch1)                       # Register the independent parameter...\n",
+    "context_meas.register_parameter(dmm.v1, setpoints=(dac.ch1,))  # ...then register the dependent parameter"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "After initialising the database we create the `Experiment` object. This object contains the name of the experiment and the sample, and the path of the database. You can  use `load_or_create_experiment` to find and return an experiment with the given experiment and sample name if it already exists, or create one if not found.\n",
-    "\n"
+    "### Example measurement loop\n",
+    "\n",
+    "The QCoDeS measurement module provides a context manager for registering parameters to measure and store results. Within the context manager, measured data is periodically saved to the database as a background process.\n",
+    "\n",
+    "To conduct a simple measurement, we can create a simple loop inside the context manager which will control the instruments, acquire data, and store the results. \n",
+    "\n",
+    "> This is the a more user-configurable approach for acquiring data in QCoDeS.\n"
```

Comment:
I think it would be good to put a link to the performing measurement ... notebook here to show more advanced stuff

Reply:
done